lib/inc/internal/facts/resolvers/virtualization_resolver.hpp

```diff
@@ -10,6 +10,36 @@
 namespace facter { namespace facts { namespace resolvers {
 
     /**
+     *  Represents cloud data.
+     */
+    struct cloud_
```

Comment:
Any reason to use `cloud_` rather than `cloud`? We tend to use an underscore suffix for private data members.